proto/storage/compliance_operator_v2.proto

```diff
@@ -135,6 +135,18 @@ message ComplianceOperatorClusterScanConfigStatus {
   string cluster_name = 5;
 }
 
+// Next Tag: 8
+message ComplianceOperatorBenchmarkV2 {
+  string id = 1; // @gotags: sql:"pk,id,type(uuid)"
+  string name = 2; // @gotags: search:"Compliance Benchmark Name,hidden"
+  string version = 3;
+  string description = 4;
+  // Example: https://control.compliance.openshift.io/CIS-OCP
+  repeated string profile_annotation = 5;
+  string provider = 6;
+  repeated string profile_names = 7;
```

Comment:
To make this searchable and joinable it would need to be something like this:

```suggestion
    message Profile {
    string profile_name = 1; // @gotags: search:"Compliance Profile Name" sql:"fk(ComplianceOperatorProfileV2:name),no-fk-constraint"
    string profile_version = 2; // @gotags: search:"Compliance Profile Version" sql:"fk(ComplianceOperatorProfileV2:profile_version),no-fk-constraint"
  }
  repeated Profile profiles = 10;
```

Reply:
Done